main/src/library/File.flix

```diff
@@ -840,4 +840,87 @@ namespace File {
                 Err(getMessage(ex))
         }
 
+    ///
+    /// Moves a file or directory from path `src` to path `dst`.
+    ///
+    /// Returns `Ok(())` if `src` was moved, and `dst` did not already exist.
+    /// Returns `Err(msg)` if `src` was not moved, because `dst` already exists or is a subpath of `src`, or an I/O error occurred.
```

Comment:
Is this sentence too long?

Reply:
I think it's ok :)

But if you want, you can separate it into points, like this;
```
Returns `Err(msg)` if `src` was not moved because:
- `dst` already exists, or
- `dst` is a subpath of `src`, or
- an I/O error occurred.